app/vmalert/rule/alerting.go

```diff
@@ -36,13 +35,13 @@ type AlertingRule struct {
 
 	q datasource.Querier
 
-	alertsMu sync.RWMutex
+	AlertsMu sync.RWMutex
```

Comment:
mutex should be private. Instead, provide thread safe public method which will internally acquire and release the lock

Reply:
Noted, fixed!